verification-annotations/src/tests.rs

```diff
@@ -85,6 +86,110 @@ fn t5() {
     verifier::assert_ne!(a, a+1);
 }
 
+#[cfg_attr(not(feature = "verifier-crux"), test)]
+#[cfg_attr(feature = "verifier-crux", crux_test)]
+fn bytes() {
+    let a = verifier::verifier_nondet_bytes(8);
+    for i in a.iter() {
```

Comment:
`for i in &a {` will call `a.iter()`. Note the `&`, otherwise it calls `a.into_iter()` which moves `a` and you can no longer use it.